test/jdk/com/sun/net/httpserver/simpleserver/SimpleFileServerTest.java

```diff
@@ -52,6 +52,7 @@
 import jdk.test.lib.Platform;
 import jdk.test.lib.net.URIBuilder;
 import jdk.test.lib.util.FileUtils;
+import jtreg.SkippedException;
```

Comment:
You should use the testng exception instead - testng doesn't recognize the jtreg exception